client/src/Client/pages/Passenger/OrderRide/OrderRide.tsx

```diff
@@ -311,6 +325,82 @@ const OrderRide = () => {
           )}
         </FormControl>
 
+        <FormControl>
+          <DatePicker
+            label="תאריך איסוף מבוקש"
+            defaultValue={dayjs()}
+            maxDate={dayjs().add(3, 'day')}
+            disablePast
+            onChange={(date) => {
+              setValue('ride.pickupDateTime', date ? date.toDate() : undefined);
+            }}
+            format="YYYY-MM-DD"
```

Comment:
@aviv-taori Same comments as in client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx